lib/nsxv.py

```diff
@@ -209,6 +209,11 @@ def __str__(self):
     source_addr = []
     destination_addr = []
 
+    source_ipv4_address = False
+    source_ipv6_address = False
+    dest_ipv4_address = False
+    dest_ipv6_address = False
```

Comment:
So I think this code can be made a bit cleaner. Instead of defining these boolean that are used for the flow control at line 256, you could instead create two more lists and rename the existing lists.
source_v4_addr
source_v6_addr
dest_v4_addr
dest_v6_addr

 In addition an optimization could occur that if after you finished source section, if you only v4 were used, you can skip the v6 for dest loop since they would be dropped anyways, but that's totally optional and not sure which is more performant.

And in the loop on 217 where it goes through each af_list, just use the correct list for the storage of the values.

Following that where the flow control starts at 256, you can just the size or status of the lists created as an implicit way to determine if their was v6 address or not. Then you can just copy the values to the final array, or just join the arrays together.